DSL/com.larsreimann.safeds/model/SafeDS.ecore

```diff
@@ -155,6 +156,9 @@
     <eClassifiers xsi:type="ecore:EClass" name="SdsGoalReference" eSuperTypes="#//SdsAbstractGoalExpression">
         <eStructuralFeatures xsi:type="ecore:EReference" name="declaration" eType="#//SdsAbstractDeclaration" />
     </eClassifiers>
+    <eClassifiers xsi:type="ecore:EClass" name="SdsSchemaEffectReference" eSuperTypes="#//SdsAbstractGoalExpression">
```

Comment:
Since this PR is about atomic schema effect, I'd prefer calling this "SdsAtomicSchemaEffect".

Reply:
Done !